docs/topics/settings.rst

```diff
@@ -749,6 +749,14 @@ The Feed Temp dir allows you to set a custom folder to save crawler
 temporary files before uploading with :ref:`FTP feed storage <topics-feed-storage-ftp>` and
 :ref:`Amazon S3 <topics-feed-storage-s3>`.
 
+.. setting:: FEED_STORAGE_GCS_ACL
+
+FEED_STORAGE_GCS_ACL
+--------------------
+
+The Access Control List (ACL) used when storing items to :ref:`Google Cloud Storage <topics-feed-storage-gcs>`.
+For more information on how to set this value, please refer to `Google Cloud documentation <https://cloud.google.com/storage/docs/access-control/lists>`_.
```

Comment:
I had a quick look at the linked documentation, and when using [predefined ACLs](https://cloud.google.com/storage/docs/access-control/lists#predefined-acl), I would not be sure whether the value defined here would have to be the one from the `JSON API` column, the one from the `XML API/gsutil`, or either. What would you think about clarifying it here?

Reply:
Just updated to mention the JSON API